### ISSUE TITLE
Area beacon for mining.

### ISSUE THREAD

**Issue by M0dEx** [Nanite-Construction-System/NaniteConstructionSystem#133]
Area beacon for mining (when you want to mine out a part of an asteroid to build your base, etc.)

**Comment by fank** [comment]
The nanite ore detector was designed to be the new beacon for mining.
Because mining is a very load intensive task and will also provide a valuable output in game.
We had a mining beacon in 1.x which as reworked into a upgrade able ore detector in 2.x

**Comment by fank** [comment]
The refactor issue #25

**Comment by fank** [comment]
And if the idea behind was to provide something with a larger range, the plan we had was to have a big drill for large area mining #47 (idea was to extract only ores, but could be used to remove all voxels)

**Comment by M0dEx** [comment]
> And if the idea behind was to provide something with a larger range, the plan we had was to have a big drill for large area mining #47 (idea was to extract only ores, but could be used to remove all voxels)

Yeah, I miss something with which I could just prepare an area of an asteroid for base building (where you could just specify a precisely measured area that it would completely mine out.)
